holoviews/core/spaces.py

```diff
@@ -402,6 +402,13 @@ class Callable(param.Parameterized):
     returned value based on the arguments to the function and the
     state of all streams on its inputs, to avoid calling the function
     unnecessarily.
+
+    A Callable may also specify a stream_mapping which allows
+    specifying which objects to attached linked streams to on
+    callbacks which return composite objects like (Nd)Layout and
+    GridSpace objects. The mapping should map between an integer index
+    or a type[.group][.label] specification and lists of streams
+    matching the object.
     """
 
     callable_function = param.Callable(default=lambda x: x, doc="""
```

Comment:
This doesn't have to happen in this PR but I'm strongly considering having this renamed to simply ``callable``. It would be a parameter/attribute and so wouldn't shadow the ``callable`` builtin.